l10n/bundle.l10n.de.json

```diff
@@ -4,6 +4,7 @@
   "Cancelling job...": "Abbruch des Jobs...",
   "Cannot call self on ComputeSession with no id": "Kann self nicht für ComputeSessions ohne ID aufrufen",
   "Cannot call self on object with no id": "Kann self für ein Object ohne ID nicht aufrufen",
+  "Cannot find file: {file}": "Cannot find file: {file}",
```

Comment:
Hey @Criptic . Do you mind providing translations for these new terms when you have some time?